eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -122,40 +116,33 @@ jobs:
       - script: |
           node common/scripts/install-run-rush.js install
         displayName: "Install dependencies"
-        condition: eq(variables['DependencyVersion'],'repo')
 
       # Option "-p max" ensures parallelism is set to the number of cores on all platforms, which improves build times.
       # The default on Windows is "cores - 1" (microsoft/rushstack#436).
       - script: |
           node common/scripts/install-run-rush.js build -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build libraries"
-        condition: eq(variables['DependencyVersion'],'repo')
 
       # Option "-p max" ensures parallelism is set to the number of cores on all platforms, which improves build times.
       # The default on Windows is "cores - 1" (microsoft/rushstack#436).
       - script: |
           node common/scripts/install-run-rush.js build:test -t "${{parameters.PackageName}}" --verbose -p max
         displayName: "Build test assets"
-        condition: eq(variables['DependencyVersion'],'repo')
+        condition: not(variables['DependencyVersion'])
```

Comment:
I'm not 100% this will work in all cases, I'm not sure what it it resolves to when empty or contains a string. This might need to be `ne(variables['DependencyVersion'], '')`. You should also add the succeeded() to this function.